rs/execution_environment/src/execution_environment.rs

```diff
@@ -3190,12 +3189,13 @@ impl ExecutionEnvironment {
     ) -> Result<(), UserError> {
         let args = VetKdDeriveKeyArgs::decode(payload)?;
         let key_id = MasterPublicKeyId::VetKd(args.key_id.clone());
+        let nidkg_key_id = NiDkgMasterPublicKeyId::VetKd(args.key_id.clone());
```

Comment:
`key_id` is now only used in logs, right? I think we should change all those to use `nidkg_key_id` and remove `key_id`.